warehouse/models/intermediate/gtfs/gtfs_quality/int_gtfs_quality__demand_responsive_routes_match.sql

```diff
@@ -0,0 +1,27 @@
+WITH
+
+idx AS (
+    SELECT * FROM {{ ref('int_gtfs_quality__gtfs_service_data_guideline_index') }}
```

Comment:
Another sorry I should have caught before -- I think that this one only applies to schedule feeds. 
So unfortunately we may need a separate `gtfs_service_data_schedule` index?